app/views/blog/posts/2010-03-10-bundles.markdown

```diff
@@ -69,7 +69,7 @@ via a bundle on a usb stick or email.
 
 First we need to determine the range of commits we want to include in the bundle.
 The easiest way would have been to drop a branch when we started, so we could
-say `start_branch..master` or `master ^start_branch`, but if we didn't we can
+say `master..start_branch` or `master ^start_branch`, but if we didn't we can
```

Comment:
I'm confused. I thought the article was claiming that you would set `start_branch` ahead of time, so that you could then later use it as a fence-post for seeing what is new on master. In which case `start_branch..master` would be the right thing.

If it does need to be flipped, then the second bit would need it too (i.e., `master..start_branch` is `^master start_branch`).

Maybe using a different name than `start_branch` would help?

Reply:
I assumed that after cloning the bundle into master, he created a `start_branch` where he would do the work. not sure if that's what you are saying, or the opposite

Reply:
I think the "would have been..." is implying that we created `start_branch` long ago (perhaps right after cloning) and there has since been work on master that we want to send in the bundle. If that is the case, using a tag would probably be a little more straightforward.

Reply:
I see your point, and it makes sense. Should we discard the PR then?

Reply:
> Should we discard the PR then?

I think so, but I'm open to @tmornini pointing out something we've missed.